.releaserc

```diff
@@ -1,15 +1,18 @@
 {
   "branch": "master",
-  "verifyConditions": [],
+  "verifyConditions": ["@semantic-release/changelog", "@semantic-release/github"],
+  "debug": true,
   "prepare": [
+    "@semantic-release/changelog",
```

Comment:
the "exec" step needs to be done first, then changelog, then git.   You can see the python-core file as an example.